scripts/kconfig/menuconfig.py

```diff
@@ -124,6 +124,25 @@ def _init_styles():
     global _INFO_BOT_SEP_STYLE
     global _INFO_HELP_STYLE
 
+    if sys.platform.startswith('win'):
+        COLOR_BRIGHT_BLACK   = 8
+        COLOR_BRIGHT_BLUE    = 9
+        COLOR_BRIGHT_GREEN   = 10
+        COLOR_BRIGHT_CYAN    = 11
+        COLOR_BRIGHT_RED     = 12
+        COLOR_BRIGHT_MAGENTA = 13
+        COLOR_BRIGHT_YELLOW  = 14
+        COLOR_BRIGHT_WHITE   = 15
```

Comment:
Where do these values come from? A comment would be nice, that also gives an overview of the issue.

This is probably an RFC, but I got curious anyway.

Reply:
on the Windows Command prompt:

```
color /?
```